cmd/fetch_ca_cert/fetch_ca_cert.go

```diff
@@ -188,6 +188,7 @@ func main() {
 
 	client, err := privateca.NewCertificateAuthorityClient(context.Background())
 	if err != nil {
+		client.Close()
```

Comment:
We probably want to close the client whether this function returns an error or not. Maybe put this as a defer statement above as well?

Reply:
Good catch!